.github/workflows/mirror-certs.yaml

```diff
@@ -12,8 +12,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: Download certificates
         run: |
-          wget --no-parent -nd -r -N --retry-connrefused --wait 5 --random-wait 'http://aia.pki.va.gov/PKI/AIA/VA/'
-          rm index.html
+          wget -r -N --wait 5 --random-wait --retry-connrefused -np -nH --cut-dirs 5 -A.cer http://crl.pki.va.gov/PKI/AIA/VA/ -P "${ANCHORS}"
```

Comment:
`--wait 5 --random-wait --retry-connrefused` keeping these commands due to site flakiness

Keeping `N` to only get certs updated, rather than redownload